.azure-pipelines/MacOS-CI.yml

```diff
@@ -33,7 +54,7 @@ jobs:
       export ONNX_BUILD_TESTS=1
       export DEBUG=${onnx_debug}
       export ONNX_ML=${onnx_ml}
-      export CMAKE_ARGS="-DONNX_USE_LITE_PROTO=ON -DONNXIFI_DUMMY_BACKEND=ON"
+      export CMAKE_ARGS=-DONNX_USE_LITE_PROTO=${onnx_lite} -DONNXIFI_DUMMY_BACKEND=ON
```

Comment:
quotation marks are required otherwise it does not recognize all the args. This is from MacOS CI for this PR
/Users/runner/work/_temp/d95fff7f-6a8a-44fd-a5d0-8d57263db9dc.sh: line 10: export: `-DONNXIFI_DUMMY_BACKEND=ON': not a valid identifier

Reply:
Thanks! I've fixed this in the latest update.